security/rbac/scripts/enable-rbac-ksql-server.sh

```diff
@@ -0,0 +1,167 @@
+
+echo -e "\n# Grant principal User:${ADMIN_KSQL} the DeveloperManage role to Cluster:kafka-cluster"
+echo "confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role DeveloperManage --resource Cluster:kafka-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role DeveloperManage --resource Cluster:kafka-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Grant principal User:${ADMIN_KSQL} the ResourceOwner role to Group:_confluent-ksql-${KSQL_SERVICE_ID} prefix"
+echo "confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role ResourceOwner --resource Group:_confluent-ksql-${KSQL_SERVICE_ID} --prefix --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role ResourceOwner --resource Group:_confluent-ksql-${KSQL_SERVICE_ID} --prefix --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Grant principal User:${ADMIN_KSQL} the ResourceOwner role to Topic:${KSQL_SERVICE_ID}ksql_processing_log"
+echo "confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role ResourceOwner --resource Topic:${KSQL_SERVICE_ID}ksql_processing_log --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role ResourceOwner --resource Topic:${KSQL_SERVICE_ID}ksql_processing_log --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Grant principal User:${ADMIN_KSQL} the DeveloperRead role to Topic:$DATA_TOPIC"
+echo "confluent iam rolebinding create --principal User:$user --role DeveloperRead --resource Topic:$DATA_TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:${ADMIN_KSQL} --role DeveloperRead --resource Topic:$DATA_TOPIC --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# List the role bindings for the principal User:$ADMIN_KSQL"
+echo "confluent iam rolebinding list --principal User:$ADMIN_KSQL --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding list --principal User:$ADMIN_KSQL --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# List the role bindings for the principal User:$ADMIN_KSQL to the KSQL cluster"
+echo "confluent iam rolebinding list --principal User:$ADMIN_KSQL --kafka-cluster-id $KAFKA_CLUSTER_ID --ksql-cluster-id $KSQL_SERVICE_ID"
+confluent iam rolebinding list --principal User:$ADMIN_KSQL --kafka-cluster-id $KAFKA_CLUSTER_ID --ksql-cluster-id $KSQL_SERVICE_ID
+
+echo -e "\n# KSQL CLI: list topics and print messages from topic $DATA_TOPIC"
+echo "ksql -u $KSQL_USER -p ksqlclient1 http://localhost:8088"
+echo
+ksql -u $KSQL_USER -p ksqlclient1 http://localhost:8088 <<EOF
+list topics;
+PRINT '$DATA_TOPIC' FROM BEGINNING LIMIT 10;
+exit ;
+EOF
+
+echo -e "\n# KSQL CLI: create a new stream and select * from that stream"
```

Comment:
iirc, creating stream requires different permissions from creating a table. Would be a good self-test to create a table here too, this would mean permissions are setup correctly.

Reply:
@stan-confluent 👍